src/Symfony/Component/HttpFoundation/ParameterBag.php

```diff
@@ -202,4 +202,22 @@ public function count()
     {
         return \count($this->parameters);
     }
+
+    /**
+     * Returns the parameter value converted to array.
+     */
+    public function getJsonArray(string $key, ?array $default = []): array
+    {
+        $value = $this->get($key, $default);
+
+        if (\is_string($value)) {
+            $value = json_decode($value, true);
+        }
+
+        if (\is_object($value)) {
+            $value = (array) $value;
```

Comment:
I did not expect this is a method that is supposed to decode JSON, tbh. What’s the use-case?

Reply:
I just have tried to fix any coming unexpected errors in advance. :)

Reply:
Should I remove this check?